<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1497</width>
    <height>953</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>与或非加法器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>1440</width>
      <height>810</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/E:/照片/纳西妲.jpg);
border-radius:30px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>1440</width>
      <height>810</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 245);
border-radius:30px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>100</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(197, 0, 0,100);
border-radius:25px
</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="res">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>730</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);
border-radius:20px</string>
    </property>
    <property name="text">
     <string>计算结果</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Secondnum">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(203, 203, 203);
border-radius:30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>请输入一个大于等于0的整数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Firstnum">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(203, 203, 203);
border-radius:30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>请输入一个大于等于0的整数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Countbuttons">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>610</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">Countbuttons{
background-color: rgba(255, 255, 255,220);
border-radius:30px;
}
Countbuttons:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(75, 204, 191, 255), stop:1 rgba(255, 255, 255, 255));
border-radius:30px;
}</string>
    </property>
    <property name="text">
     <string>开始计算</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>461</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(229, 255, 231, 240);
border-radius:30px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="BinA">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>670</y>
      <width>424</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 252, 255);</string>
    </property>
    <property name="text">
     <string>第一个数的二进制</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="BinB">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>730</y>
      <width>424</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 252, 255);</string>
    </property>
    <property name="text">
     <string>第二个数的二进制</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Binres">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>790</y>
      <width>424</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 252, 255);</string>
    </property>
    <property name="text">
     <string>结果的二进制</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="Record">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>560</y>
      <width>431</width>
      <height>321</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>84</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>本位A</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>本位B</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>进位in</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>本位out</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>进位out</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="ptr">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>600</y>
      <width>424</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>15</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Adder">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>861</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/F:/素材/图片/全加器.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="StandA1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>601</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/arr/全加器/本位A为1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="StandA0">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>601</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/arr/全加器/本位A为0.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="StandB1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>601</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/arr/全加器/本位B为1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="StandB0">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>601</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/arr/全加器/本位B为0.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Carry1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>861</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/arr/全加器/进位为1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Carry0">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>861</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/arr/全加器/进位为0.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Standxor1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>601</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/arr/全加器/本位异或为1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Standxor0">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>601</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/arr/全加器/本位异或为0.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Standout1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>601</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/arr/全加器/本位结果为1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Standout0">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>601</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/arr/全加器/本位结果为0.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Carryup1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>861</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/arr/全加器/进位前值A为1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Carryup0">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>861</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/arr/全加器/进位前值A为0.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Carrydown1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>861</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/arr/全加器/进位前值B为1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Carrydown0">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>861</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/arr/全加器/进位前值B为0.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Carryout1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>861</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/arr/全加器/进位OUT为1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Carryout0">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>861</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/arr/全加器/进位OUT为0.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>与或非全加器</string>
    </property>
   </widget>
   <widget class="QLabel" name="NumstandA">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>140</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="NumstandB">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>190</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Numcarry">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>270</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Numstandout">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>190</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Numcarryout">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>390</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Sleeptime">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeSleep">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>520</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>更改延时</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>延时效果：</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>Firstnum</zorder>
   <zorder>Countbuttons</zorder>
   <zorder>Secondnum</zorder>
   <zorder>res</zorder>
   <zorder>BinA</zorder>
   <zorder>BinB</zorder>
   <zorder>Binres</zorder>
   <zorder>Record</zorder>
   <zorder>ptr</zorder>
   <zorder>Adder</zorder>
   <zorder>StandA1</zorder>
   <zorder>pushButton</zorder>
   <zorder>StandA0</zorder>
   <zorder>StandB1</zorder>
   <zorder>StandB0</zorder>
   <zorder>Carry1</zorder>
   <zorder>Carry0</zorder>
   <zorder>Standxor1</zorder>
   <zorder>Standxor0</zorder>
   <zorder>Standout1</zorder>
   <zorder>Standout0</zorder>
   <zorder>Carryup1</zorder>
   <zorder>Carryup0</zorder>
   <zorder>Carrydown1</zorder>
   <zorder>Carrydown0</zorder>
   <zorder>Carryout1</zorder>
   <zorder>Carryout0</zorder>
   <zorder>label_4</zorder>
   <zorder>NumstandA</zorder>
   <zorder>NumstandB</zorder>
   <zorder>Numcarry</zorder>
   <zorder>Numstandout</zorder>
   <zorder>Numcarryout</zorder>
   <zorder>Sleeptime</zorder>
   <zorder>ChangeSleep</zorder>
   <zorder>label_5</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="q.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>939</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>1003</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
